auth/roles.yaml

```diff
@@ -400,3 +412,95 @@
     - name: rulebundles
       verbs:
         - get
+    - name: migrationwaves
+      verbs:
+        - get
+- role: tackle-project-manager
```

Comment:
hint: this is a new persona that only has write permissions on migration waves. see https://github.com/konveyor/enhancements/tree/master/enhancements/migration-waves